administrator-manual/en/base_system2.rst

```diff
@@ -261,3 +261,49 @@ The following pages are always accessible to all users:
 * dashboard
 * applications
 * terminal
+
+Two-factor authentication (2FA)
+===============================
+
+Two-factor authentication (2FA) is an extra layer of security required to access the new Server Manager.
```

Comment:
Two-factor authentication (2FA) is an (optional) extra layer of security...

Reply:
It's not optional once enforced.